XenAdmin/Dialogs/ScheduledSnapshots/ScheduledSnapshotsDialog.cs

```diff
@@ -535,16 +520,16 @@ private void RefreshHistoryLabel()
 
         private class HistoryRow : DataGridViewRow
         {
-            private DataGridViewImageCell _expand = new DataGridViewImageCell();
-            private DataGridViewTextAndImageCell _result = new DataGridViewTextAndImageCell();
-            private DataGridViewTextBoxCell _dateTime = new DataGridViewTextBoxCell();
-            private DataGridViewTextBoxCell _description = new DataGridViewTextBoxCell();
+            private readonly DataGridViewImageCell _expand = new DataGridViewImageCell();
+            private readonly DataGridViewImageCell _image = new DataGridViewImageCell();

```

Comment:
With the switch to an image cell, we lost the ability to sort on that column (I don't know if it's possible to sort on Image cells)